src/Compilers/CSharp/Portable/Binder/SwitchBinder.cs

```diff
@@ -101,6 +101,18 @@ override protected ImmutableArray<LocalSymbol> BuildLocals()
             return builder.ToImmutableAndFree();
         }
 
+        protected override ImmutableArray<LocalFunctionSymbol> BuildLocalFunctions()
+        {
+            var builder = ArrayBuilder<LocalFunctionSymbol>.GetInstance();
+
+            foreach (var section in _switchSyntax.Sections)
+            {
+                builder.AddRange(BuildLocalFunctions(section.Statements));
```

Comment:
You're right. Ignore my comment.

We'll have to talk about definite assignment for local functions, but not today.